### PR TITLE
Sp 21 Time Restrict and More Info - Dept Tour

### PR DESCRIPTION
**Pull request by ochan1**

### CODE REVIEW

**Comment by ochan1** [comment]
All fixes are front end based

File: app/assets/javascripts/datetimepicker.js.erb [hunk 1]
```diff
@@ -5,8 +5,13 @@ $(document).ready(function() {
     // Only show weekdays, since tours are not offered on weekends
     beforeShowDay: function(date) {
       var td = date.getDay();
+      // OLD
       // Don't allow Saturday (6) or Sunday (0) for tours
-      var selectable = (td != 0 && td != 6);
+      // var selectable = (td != 0 && td != 6);
```
Comment (sammaher1) [inline app/assets/javascripts/datetimepicker.js.erb:10]: Limited to Thursday and Saturday, good.

**Comment by ochan1** [comment]
Also, I should mention that you won't be able to submit due to the Captcha, but only the frontend stuff is modified, so it shouldn't affect how the form is submitted

**Review by sammaher1 — APPROVED** [review]
Works well, tested it. Prevents incorrect date from being selected, though limiting incorrect time remains to be done. Approved.

**Comment by ochan1** [comment]
For future reference for anyone who sees this:

Due to limitations of the library used for the datetimepicker, I can't limit the selection of the time
However, submitting the form requires the correct time to be submitted and an error will be shown

Hopefully, this library will allow this down the line or an alternate is made (like one date field and one time field)

Thank you, Sam

**Pull request merged**